src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrBinaryFileScanner.cs

```diff
@@ -72,12 +73,31 @@ private async Task<IEnumerable<string>> ScanAndParseResult(string[] args, string
 
         ret.ThrowIfFailed($"Failed to scan the {repoName} repository");
 
-        // We want to exlude empty files from the list. These empty files still have a few chracters that we want to ignore
+        // Git marks UTF 16 text files as binary, we want to exclude these
```

Comment:
```suggestion
        // Git evaluates UTF-16 text files as binary, we want to exclude these
```

Reply:
I'd put the comment above the line that does this

Reply:
done